core/state_processor.go

```diff
@@ -78,7 +79,11 @@ func (p *StateProcessor) Process(block *types.Block, parent *types.Header, state
 	)
 
 	// Configure any stateful precompiles that should go into effect during this block.
-	p.config.CheckConfigurePrecompiles(new(big.Int).SetUint64(parent.Time), block, statedb)
+	err := p.config.ConfigurePrecompiles(new(big.Int).SetUint64(parent.Time), block, statedb)
+	if err != nil {
+		log.Error("failed to configure precompiles processing block", "hash", block.Hash(), "number", block.NumberU64(), "timestamp", block.Time(), "err", err)
```

Comment:
Is this never possible to occur (since we are logging an Error)?

Reply:
we previously changed it to log an error. but if we think returning an error is sufficient we can revert this. cc: @aaronbuchwald 
https://github.com/ava-labs/subnet-evm/pull/418#discussion_r1053638786

Reply:
Yup, this should never occur. If it does occur, it indicates a critical error has occurred and the subnet is most likely borked.

In that case, if we simply return an error it will look like every block is failing verification.

If such an error does occur, it would most likely hit every node around the same time, so I think a better behavior is to log an error as opposed to having nodes panic.